lib/grpc/GrpcService.ts

```diff
@@ -63,6 +63,28 @@ const createOrder = (order: Order) => {
   return grpcOrder;
 };
 
+const createOrderFromServiceOrder = (order: ServiceOrder) => {
```

Comment:
This method isn't necessary either, you can just use the existing `createServiceOrder` in its place.